src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -328,15 +336,15 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
 
               <TokenExtras>
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-                {yieldPercent && (
-                  <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-                  </StyledText>
-                )}
               </TokenExtras>
             </TokenData>
           )}
         </TokenInfo>
+        {listItems?.length > 1 && (
+          <ZapMessageContainer onClick={listItems?.length > 1 ? openSearchList : undefined}>
+            ⚡ Or <HighlightText> select a token </HighlightText> in your wallet to ZAP
+          </ZapMessageContainer>
+        )}
```

Comment:
When there is a list of token options available, it does not necessarily mean it is a token to zap from. For example, when depositing a token from the portfolio view, you are currently able to select the vault that token is allowed to zap into:

![image](https://user-images.githubusercontent.com/18649057/169621114-08e996b0-bc40-4f3d-97d7-236f5d77e239.png)

When withdrawing, you also can select the token you want zap out into:

![image](https://user-images.githubusercontent.com/18649057/169621222-c1b4563d-7e92-4e54-93f7-4aaa6e14fee3.png)

imo, different texts should be used depending on the situation.